src/components/Navbar/Navbar.tsx

```diff
@@ -5,86 +5,124 @@ import Form from 'react-bootstrap/Form'
 import FormControl from 'react-bootstrap/FormControl'
 import NavDropdown from 'react-bootstrap/NavDropdown'
 import { Button } from '../Button'
-import { NavLink, Brand, NavLinkElement, Search } from './interfaces'
+import { NavLink, NavIcon, NavHeader, NavLinkList, NavSearch } from './interfaces'
 
 interface Props extends React.Props<any> {
   /** Determines the navbar background color */
   bg?: string
   /** Determines the letters color. It should be combined with the background color (bg) */
   variant?: 'light' | 'dark'
   /** Determines the links names, theirs onClick methods and paths. It has children array which contain links to be used on a dropdown. */
-  navLinks: NavLink[]
-  /** Determines the hospital/clinic name to be shown at the navbar */
-  brand: Brand
-  /** Defines the properties of the search element */
-  search: Search
+  navItems: (NavIcon | NavHeader | NavLink | NavLinkList | NavSearch)[]
+  /** Defines the class of the list. */
+  className?: string
 }
 
 /**
  * Used to redirect users to the main topics.
  */
 const Navbar = (props: Props) => {
-  const { bg, variant, navLinks, brand, search } = props
+  const { bg, variant, navItems, className } = props
 
-  const getNavItems = (subLink: NavLinkElement, index: number) => (
-    <NavDropdown.Item href={subLink.href ? subLink.href : ''} key={index} onClick={subLink.onClick}>
-      {subLink.label}
+  const getNavListLink = (link: NavLink, index: number) => (
+    <NavDropdown.Item
+      className={link.className ? link.className.concat(' ', '') : ''}
```

Comment:
I don't think that we need any of the concatenation stuff here. It should be sufficient to do `className={link.className}`

Same for all of the other ones, too.